src/eterna/EternaApp.ts

```diff
@@ -38,7 +38,7 @@ import Folder from './folding/Folder';
 import RSignals from './rscript/RSignals';
 import Fonts from './util/Fonts';
 
-enum PuzzleID {
+export enum PuzzleID {
```

Comment:
Yeah, I'm not sure if it's necessary. I noticed that a puzzle id that was involved in the AuxInfo infrastructure that I updated to conform to the annotations infrastructure referenced a puzzle that was in here. To avoid magic numbers, I decided to utilize this one

Reply:
Can we edit the Drupal node to specify the annotation there?

Reply:
Having trouble visualizing the suggestion you're making

Reply:
The other annotations are stored within Drupal via the targetConditions (or as they're called on the backend, "objectives" or whaetever), right? Can't we just add it to the JSON in that field?

Reply:
oh, yes totally reasonable. can it be done within the client or will I have the interface with the CLI?

Reply:
Actually, I just realized that we'll have to keep it hard-coded because this code relies on the annotation being removed after a particular binding event. The infrastructure as it is doesn't support state-based stuff like this quite yet.